### PR TITLE
notionflux: Allow passing a filename, which is sent as dofile() stmt

### PR DESCRIPTION
**Pull request by wilhelmy**
This is useful for passing long files which don't fit into mod_notionflux's buffer.
Also some more manpage stuff.

### CODE REVIEW

File: man/notionflux.in [hunk 1]
```diff
@@ -92,6 +115,8 @@ The document
 .Dq Configuring and extending Notion with Lua
 found on the Notion homepage.
 .Pp
+.Lk https://www.lua.org/manual/LUA_REFVERSION/manual.html "The Lua LUA_REFVERSION reference manual"
```
Comment (raboof) [inline man/notionflux.in]: neat ;)

File: mod_notionflux/notionflux/notionflux.c [hunk 2]
```diff
@@ -365,32 +367,77 @@ int repl_main(void)
...
+			buf_append(buf, b);
+		} else {
+			char b[6] = {0};
+			snprintf(b, sizeof(b), "\\%d", *p);
+			buf_append(buf, b);
+		}
+	}
+
+	if (!buf_append(buf, "');--"))
+		die("Filename too long.\n");
```
Comment (raboof) [inline mod_notionflux/notionflux/notionflux.c]: pretty huge path :smile:

Comment (wilhelmy) [reply mod_notionflux/notionflux/notionflux.c]: You never know, the filename could be 1024 bytes of non-printable \127 characters :)

**Comment by knixeur** [comment]
Build on C89 failed but I think we agree on moving on to C99, once we fix the build LGTM :+1:

**Pull request merged**